incubator/crate/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: crate
+version: 0.1.0
+description: A Helm chart for CrateDB
+appVersion: 3.1.1
+home: https://crate.io
+icon: https://crate.io/wp-content/themes/webflow/images/logo-crate.png
```

Comment:
Would it be possible to add `sources` to the helm chart? Specifically:
```
sources:
- https://github.com/crate/crate
```

Reply:
got it!